src/plugins/discover/kibana.json

```diff
@@ -16,5 +16,11 @@
     "indexPatternFieldEditor"
   ],
   "optionalPlugins": ["home", "share", "usageCollection"],
-  "requiredBundles": ["kibanaUtils", "home", "kibanaReact"]
+  "requiredBundles": ["kibanaUtils", "home", "kibanaReact"],
+  "owner": {
+    "name": "Kibana App",
+    "githubTeam": "kibana-app"
+  
```

Comment:
:art:  Is this empty line having a purpose (in life and otherwise)?

Reply:
Just because I wanted to see you pain ;) thx, wouldn't have happened if I had edited the PR locally :)